lib/timeline/frame_flame_chart.dart

```diff
@@ -71,9 +71,7 @@ class FrameFlameChart extends CoreElement {
 
     int maxRow = 0;
 
-    Function drawRecursively;
-
-    drawRecursively = (TimelineThreadEvent event, int row) {
+    void _drawRecursively(TimelineThreadEvent event, int row) {
```

Comment:
no need to make this name private as it is defined within another method so can't possibly be used outside the class.

Reply:
whoops good catch. fixed.